src/components/ha-attributes.ts

```diff
@@ -10,6 +10,7 @@ import {
 import { HassEntity } from "home-assistant-js-websocket";
 
 import hassAttributeUtil from "../util/hass-attributes-util";
+import { safeDump } from "js-yaml";
```

Comment:
Can we import this async? We won't need it most of the time and we import the more info dialogs pretty early on.

Reply:
We can use memoizeOne and a promise to import + render combined with lit rendering output of promise.

Reply:
only care about this component because it's used in more info. For dev tool I don't care